packages/math/src/BoundingFrustum.ts

```diff
@@ -143,6 +144,33 @@ export class BoundingFrustum implements IClone<BoundingFrustum>, ICopy<BoundingF
     return CollisionUtil.frustumContainsSphere(this, sphere) !== ContainmentType.Disjoint;
   }
 
+  /**
+   * Get whether or not a point is in frustum.
+   * @param point - The point for testing
+   * @returns True if the point is in this frustum, false otherwise
+  */
+  public containsPoint(point: Vector3): boolean {
+    if (Plane.distanceToPoint(this.top, point) < 0) {
+      return false;
+    }
+    if (Plane.distanceToPoint(this.right, point) < 0) {
+      return false;
+    }
+    if (Plane.distanceToPoint(this.bottom, point) < 0) {
+      return false;
+    }
+    if (Plane.distanceToPoint(this.left, point) < 0) {
+      return false;
+    }
+    if (Plane.distanceToPoint(this.near, point) < 0) {
+      return false;
+    }
+    if (Plane.distanceToPoint(this.far, point) < 0) {
+      return false;
+    }
+    return true;
+  }
```

Comment:
Add this in `CollisionUtil` and named `frustumContainspoint`, all determining intersections, containment, etc fun is in `CollisionUtil`

Reply:
handled